src/Smp.Web/ClientApp/src/app/messages/messages.component.scss

```diff
@@ -2,11 +2,31 @@
 @import '~sass/header-partial';
 
 .messages-body {
-  height: 100vh;
+  padding: 4rem 0 0 0;
```

Comment:
This padding has been added because the existing css didn't properly account for the header, and so the grid would start from the very top of the page.